domains/xandaah.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "xandaaah",
+        "email": "",
+        "discord": "726910228289028127",
+        "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.oxrqqSqk9XxZnJWHNR5pvh-uugiwuZObvtdKEK13Ukiuv_JoIVoADXbyLKQsWMbmC5RkNae5ynPvf7hCEljvIes9SPv7RAmkBXlJ62dSgF4pv500KKYhw3rYfvLH42f-Q122hxEDqrCEE-ipcfBJKuLw6XZm-RJBRZWOqZ_qcWlMmt1lODds5zPfccXuaUE2_CzN1vEfxm_gFeLRGPAwavy3Iu-03wCoE7GCljIdL4fTWTC5B6CoKYlT8WBtbcTsbIYKPvx6jaZcoVyK9gyAlX4MzUT1bGn9CfsBeJFPsr2fcrVV7WUlTKwGb7m0dE7i0lsVs-IDHIc0Fj75n6lCFA.4RWb42LiXSLOcR862EbpwA.7zo3KErqOjZC9VbtZ6KxQ6DW6M8gAwJCE2Vq8oEulH8TlThXAi0qTEdLLGxJoyJvlR_0ACfcVlJnPsOUx5fDnOJd2rG9SgqTH6uqLS1UUzc.gWnPDXFPz_YGFAnBd5GuIA"
+    },
+    "record": {
+        "CNAME": "xandaaah.github.io"
```

Comment:
404 GitHub Page. Make sure to have an actual site with content. See docs: https://docs.is-a.dev

Reply:
I just added a basic home page.